pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1965,7 +1965,7 @@ func checkPerPodSchedulingMetrics(name string, t *testing.T, pInfo *framework.Qu
 	if pInfo.Attempts != wantAttempts {
 		t.Errorf("[%s] Pod schedule attempt unexpected, got %v, want %v", name, pInfo.Attempts, wantAttempts)
 	}
-	if pInfo.InitialAttemptTimestamp != wantInitialAttemptTs {
+	if pInfo.InitialAttemptTimestamp != nil && *pInfo.InitialAttemptTimestamp != wantInitialAttemptTs {
```

Comment:
do we need this nil check? I'd expect to keep the old logic.

Reply:
I mean this is a testing utility, so not imposing this nil check may expose some corner case that we haven't caught, with minimum cost. (running the UTs without the nil check also works)